### DISCUSSION TITLE
Unable to run rosenbrock.in example with Dakota 6.17.0 on Windows10.

### DISCUSSION THREAD

**Discussion by mbairdphys** [snl-dakota/dakota#33] [Q&A]
Unable to run modified rosenbrock.in example with Dakota 6.17.0 on Windows10 (updated).  The file runs without issue on my Linux installation.  Both are the downloaded CLI binaries from the Dakota website.  File attached.  No error message were generated, but output file also attached.

Thanks all!
[rosenbrock.txt](https://github.com/snl-dakota/dakota/files/11222914/rosenbrock.txt)
[rosenbrockout.txt](https://github.com/snl-dakota/dakota/files/11222921/rosenbrockout.txt)

**Comment by jadamstephens** [comment]
Something quick you can try is switching the interface type from `fork` to `system`. My memory is a little fuzzy on this, but I believe we've seen issues in the past with using `fork` to run `.bat` scripts on Windows.

**Reply by mbairdphys** [reply]
@jadamstephens Thank you for the quick response, unfortunately it did not change anything.

to be specific I changed:

interface
   analysis_drivers = 'C:\Users\inb\code\WARtimE\test\..\entry.bat dakota_driver.py'
   system
   batch
     size 2
   work_directory
     named 'wrkdir'
   directory_tag
   directory_save
   copy_files = "dakota_driver.py"
                "rosenbrock.tmpl"
                "rosenbrock.drive"
   parameters_file = 'params.in'
   results_file = 'results.out'
   file_save file_tag
   
   Thank you for your time!

**Reply by jadamstephens** [reply]
What happens if you change into a work directory and run, at the command line, `C:\Users\inb\code\WARtimE\test\..\entry.bat dakota_driver.py params.in.1 results.out.1`?
